### PR TITLE
added info about scripted way of restoring

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2938

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-antivirus/restore-quarantined-files-windows-defender-antivirus.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ If Windows Defender Antivirus is configured to detect and remediate threats on y
 3. Under **Quarantined threats**, click **See full history**.
 4. Click an item you want to keep, then click **Restore**. (If you prefer to remove the item, you can click **Remove**.)
 
+> [!NOTE]
+> You can also use the dedicated command-line tool [mpcmdrun.exe](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/command-line-arguments-windows-defender-antivirus) to restore quarantined files in Windows Defender AV.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-antivirus/restore-quarantined-files-windows-defender-antivirus.md]:
```suggestion
> You can also use the dedicated command-line tool [mpcmdrun.exe](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-antivirus/command-line-arguments-windows-defender-antivirus) to restore quarantined files in Windows Defender AV.
```

**Comment by tseward** [comment]
I'm OK with this once the locale is removed from the URL.

**Comment by get-itips** [comment]
ok with @nenonix and @tseward

**Comment by dariomws306** [comment]
I am ok with the suggestions too.

**Comment by e0i** [comment]
@AndreaBarr The PR can now be merged. Thanks.

**Comment by e0i** [comment]
@AndreaBarr The PR is ready to be merged. Thanks.

**Pull request merged**